File: card-service/src/main/java/br/com/mylist/dao/entities/Card.java
package br.com.mylist.dao.entities;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import lombok.Data;

@Entity
@Data
public class Card {

	@Id
	@GeneratedValue
	private Long id;
	private String name;
	private String description;
	private String authorName;
}

File: microservices/webapp-mylist/src/app/shared/model/responses/Card.ts
export class Card {
    id : string;
	name: string;
	authorName: string;
	description: string;

	constructor(formCardName: string, formCardAuthorName: string, formCardDescription: string) {
		this.name = formCardName;
		this.authorName = formCardAuthorName;
		this.description = formCardDescription
	}
}